draft-ietf-jsonpath-base.md

```diff
@@ -1097,21 +1097,22 @@ Some examples:
 
 ## Normalized Paths
 
-A Normalized Path identifies a node by providing a query that results in exactly that node. For example,
+A Normalized Path is a JSONPath which identifies a node by providing a query that results in exactly that node. For example,
```

Comment:
```suggestion
A Normalized Path is a JSONPath that identifies a node by providing a query that results in exactly that node. For example,
```

Reply:
Ok. Must get the hang of writing English some time.